homeassistant/helpers/template.py

```diff
@@ -221,6 +224,11 @@ def _false(arg: str) -> bool:
     return False
 
 
+@lru_cache(maxsize=EVAL_CACHE_SIZE)
+def _cached_literal_eval(result: str) -> Any:
+    return literal_eval(result)
```

Comment:
Would this be slightly faster?
```suggestion
_cached_literal_eval = lru_cache(maxsize=EVAL_CACHE_SIZE)(literal_eval)
```